File: Java/Tests/jdi-uitest-webtests/src/main/java/com/epam/jdi/uitests/testing/unittests/pageobjects/w3cSite/FrameSection.java
package com.epam.jdi.uitests.testing.unittests.pageobjects.w3cSite;

import com.epam.jdi.uitests.web.selenium.elements.common.Button;
import com.epam.jdi.uitests.web.selenium.elements.composite.Section;
import org.openqa.selenium.support.FindBy;

/**
 * Created by Maksim_Palchevskii on 8/17/2015.
 */

public class FrameSection extends Section {
    @FindBy(tagName = "button")
    public Button label;
}

File: Python/Test/jdi_uitests_webtests/main/page_objects/w3c_site/frame_section.py
from JDI.web.selenium.elements.api_interact.find_element_by import By
from JDI.web.selenium.elements.common.button import Button
from JDI.web.selenium.elements.composite.section import Section


class FrameSection(Section):
    label_button = Button(By.css("button"))